site/docs/quickstarts/index.html

```diff
@@ -69,9 +69,9 @@ <h2 class="docs-lp">Web Application</h2>
                                 <div id="w-node-_50528029-a6dc-e4ee-a942-f491cb7ccf84-3d0acb82" class="docs-lp-link-block shadow-removed quickstart w-inline-block coming-soon"><img src="/assets/img/docs/landing/gatsby.svg" loading="lazy" alt="" class="quickstart-tech-icon">
                                     <div class="quickstart-tech-title">Gatsby • Coming Soon</div>
                                 </div>
-                                <div id="w-node-_26751e8d-0e6a-2024-0332-65299b30a8a0-3d0acb82" class="docs-lp-link-block shadow-removed quickstart w-inline-block coming-soon"><img src="/assets/img/docs/landing/dotnet-c.svg" loading="lazy" alt="" class="quickstart-tech-icon">
-                                    <div class="quickstart-tech-title">.NET • Coming Soon</div>
-                                </div>
+                                <a id="w-node-_26751e8d-0e6a-2024-0332-65299b30a8a0-3d0acb82"  href="/docs/v1/tech/tutorials/integrate-dotnet" class="docs-lp-link-block shadow-removed quickstart w-inline-block coming-soon"><img src="/assets/img/docs/landing/dotnet-c.svg" loading="lazy" alt="" class="quickstart-tech-icon">
```

Comment:
```suggestion
                                <a id="w-node-_26751e8d-0e6a-2024-0332-65299b30a8a0-3d0acb82"  href="/docs/v1/tech/tutorials/integrate-dotnet" class="docs-lp-link-block shadow-removed quickstart w-inline-block"><img src="/assets/img/docs/landing/dotnet-c.svg" loading="lazy" alt="" class="quickstart-tech-icon">
```

Need to remove the class otherwise it is grayed out.